src/status_im/common/bottom_sheet/view.cljs

```diff
@@ -106,7 +106,14 @@
                     {:transform [{:translateY translate-y}]}
                     (style/sheet insets window-height selected-item))}
            (when shell?
-             [blur/ios-view {:style style/shell-bg}])
+             [blur/ios-view
+              (cond-> {:style style/shell-bg}
+                blur-bg? (assoc :blur-radius
+                                blur-radius
+                                :blur-type
+                                :transparent
```

Comment:
FYI: I can't verify the implications of forcing the blur type and overlay color as transparent because I don't have iOS.

Reply:
Would it be okay if QA validate it, And also I am going to try to test all sheets.

Reply:
A quick design question: If the sheet with the type `shell?` requires a transparent background, does it apply to all sheets with this type? If yes, we can update the style of this style.

Reply:
Got a response from design, Every shell? sheet should be blurred.